kfs-ar/src/main/resources/edu/arizona/kfs/module/ar/spring-ar.xml

```diff
@@ -0,0 +1,100 @@
+                           http://www.springframework.org/schema/aop
+                           http://www.springframework.org/schema/aop/spring-aop-2.0.xsd">
+
+    <bean id="arModuleConfiguration" parent="arModuleConfiguration-parentBean">
+        <property name="triggerNames">
+            <list>
+            </list>
+        </property>
+    </bean>
+
+    <bean id="lockboxJob"                                       parent="unscheduledJobDescriptor">
+        <property name="steps">
+            <list>
+                <ref bean="lockboxLoadStep" />
+                <ref bean="lockboxStep" />
+            </list>
+        </property>
+        <property name="dependencies">
+            <map>
+                <entry key="nightlyOutJob" value="hardDependency" />
+            </map>
+        </property>
+    </bean>
+
+    <bean id="customerLoadJob"                                  parent="unscheduledJobDescriptor">
+        <property name="steps">
+            <list>
+                <ref bean="customerLoadStep" />
+            </list>
+        </property>
+    </bean>
+
+    <bean id="invoiceRecurrenceJob"                             parent="unscheduledJobDescriptor">
+        <property name="steps">
+            <list>
+                <ref bean="invoiceRecurrenceStep" />
+            </list>
+        </property>
+    </bean>
+
+    <bean id="customerInvoiceWriteoffBatchJob"                  parent="unscheduledJobDescriptor">
+        <property name="steps">
+            <list>
+                <ref bean="customerInvoiceWriteoffBatchStep" />
+            </list>
+        </property>
+    </bean>
+
+    <bean id="customerAgingReportNotificationJob"               parent="unscheduledJobDescriptor">
+        <property name="steps">
+            <list>
+                <ref bean="customerAgingReportNotificationStep" />
+            </list>
+        </property>
+    </bean>
+
+    <bean id="letterOfCreditJob"                                parent="unscheduledJobDescriptor">
+        <property name="steps">
+            <list>
+                <ref bean="letterOfCreditCreateAndRouteDocumentsStep" />
+            </list>
+        </property>
+    </bean>
+
+    <bean id="contractsGrantsInvoiceDocumentJob"                parent="unscheduledJobDescriptor">
+        <property name="steps">
+            <list>
+                <ref bean="contractsGrantsInvoiceDocumentBatchStep" />
+                <ref bean="contractsGrantsInvoiceRouteDocumentsStep" />
+            </list>
+        </property>
+    </bean>
+
+    <bean id="upcomingMilestoneNotificationJob"                 parent="unscheduledJobDescriptor">
+        <property name="steps">
+            <list>
+                <ref bean="upcomingMilestoneNotificationStep" />
+            </list>
+        </property>
+    </bean>
+
+    <bean id="clearContractsGrantsInvoiceDocumentErrorLogJob"   parent="unscheduledJobDescriptor">

```

Comment:
This job was already unscheduled, so no override needed. Also, please remove the extra spaces or tabs between bean id and parent on the other jobs (to improve readability).